webviz_subsurface/datainput/_fmu_input.py

```diff
@@ -23,3 +25,66 @@ def load_parameters(
 ) -> pd.DataFrame:
 
     return load_ensemble_set(ensemble_paths, ensemble_set_name).parameters
+
+
+@cache.memoize(timeout=cache.TIMEOUT)
+@webvizstore
+def get_realizations(
+    ensemble_paths: tuple, ensemble_set_name: str = "EnsembleSet"
+) -> pd.DataFrame:
+    ens_set = load_ensemble_set(ensemble_paths, ensemble_set_name)
+    df = ens_set.parameters.get(["ENSEMBLE", "REAL"])
+    df["SENSCASE"] = ens_set.parameters.get("SENSCASE")
+    df["SENSNAME"] = ens_set.parameters.get("SENSNAME")
+    df["RUNPATH"] = df.apply(
+        lambda x: ens_set[x["ENSEMBLE"]][x["REAL"]].runpath(), axis=1
+    )
+    return df.sort_values(by=["ENSEMBLE", "REAL"])
+
+
+@cache.memoize(timeout=cache.TIMEOUT)
+def find_surfaces(ensemble_paths: tuple, suffix="*.gri"):
+    """Reads surface file names stored in standard FMU format, and returns a dictionary
+    on the following format:
+    surface_property:
+        names:
+            - some_surface_name
+            - another_surface_name
+        dates:
+            - some_date
+            - another_date
+    """
+
+    files = []
+    for _, path in ensemble_paths:
+        files.extend(glob.glob(os.path.join(path, "share", "results", "maps", suffix)))
+
+    files = [os.path.basename(f.split(".gri")[0]) for f in files]
```

Comment:
This takes out the file name without file extension? See [pathlib.PurePath.stem](https://docs.python.org/3/library/pathlib.html#pathlib.PurePath.stem), if `f` is a `Path` you can maybe replace
```python
os.path.basename(f.split(".gri")[0])
```
with `f.stem`.